### PR TITLE
feat(bolt): add k9s to nix-shell

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

**Comment by AngelOnFira** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/903?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#904** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/904?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#903** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/903?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#740** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/740?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#750](https://github.com/rivet-gg/rivet/pull/750) <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/750?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#759** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/759?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#751** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/751?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#755** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/755?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#747** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/747?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#743** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/743?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#754** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/754?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#741** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/741?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @AngelOnFira and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
let's just add the alias, not the k9s dependency. trying to keep our nix deps as lightweight as possible, since that directly adds wait time for setting up a dev env.

**Comment by NathanFlurry** [comment]
@AngelOnFira can you move #904 on to #740

**Comment by NathanFlurry** [comment]
@AngelOnFira we can merge this if we remove k9s from the packages and just leave the alias. if you install it on the host system, k9s will work as expected.

**Comment by NathanFlurry** [comment]
@AngelOnFira we can merge this if we remove k9s from the packages and just leave the alias. if you install it on the host system, k9s will work as expected.

**Comment by AngelOnFira** [comment]
Oh ya, now that I'm not in a devcontainer that will be dope

File: shell.nix [hunk 1]
```diff
@@ -27,6 +27,7 @@ in
 			# Kubernetes tools
 			k3d
 			kubectl
+			k9s
 			kubernetes-helm
```
Comment (AngelOnFira) [inline shell.nix:31]:
```suggestion
			kubectl
			kubernetes-helm
```

**Review by NathanFlurry — APPROVED** [review]

**Review by MasterPtato — APPROVED** [review]

**Pull request merged**